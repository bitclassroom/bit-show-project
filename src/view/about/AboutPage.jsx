import React, { Component } from 'react'

class AboutPage extends Component {
    render() {
        return (
            <div>
                <h4>About TV Shows</h4>
                <p className="flow-text">This is some about description</p>
            </div>
        )
    }
}

export default AboutPage
