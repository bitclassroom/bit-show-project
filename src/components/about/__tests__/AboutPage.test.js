import React from 'react'
import renderer from 'react-test-renderer'
import AboutPage from '../AboutPage'

describe('AboutPage', () => {
    it('should match the snapshot', () => {
        const componentTree = renderer.create(<AboutPage />).toJSON()
        expect(componentTree).toMatchSnapshot()
    })
})
