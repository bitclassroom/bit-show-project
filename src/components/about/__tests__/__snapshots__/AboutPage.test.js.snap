// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage should match the snapshot 1`] = `
<div>
  <h4>
    About BIT Shows
  </h4>
  <p
    className="flow-text"
  >
    This is some about description
  </p>
</div>
`;
